python/cugraph/tests/utils.py

```diff
@@ -40,46 +40,61 @@
 #
 # Datasets
 #
-DATASETS_UNDIRECTED = ["../datasets/karate.csv", "../datasets/dolphins.csv"]
+
+
+RAPIDS_DATASET_ROOT_DIR = os.getenv("RAPIDS_DATASET_ROOT_DIR", "../datasets")
+
+DATASETS_UNDIRECTED = [PurePath(RAPIDS_DATASET_ROOT_DIR)/f for
+                       f in ["karate.csv", "dolphins.csv"]]
 
 DATASETS_UNDIRECTED_WEIGHTS = [
-    "../datasets/netscience.csv",
+    PurePath(RAPIDS_DATASET_ROOT_DIR)/"netscience.csv"
 ]
 
-DATASETS_UNRENUMBERED = ["../datasets/karate-disjoint.csv"]
+DATASETS_UNRENUMBERED = [PurePath(
+    RAPIDS_DATASET_ROOT_DIR)/"karate-disjoint.csv"
+]
 
-DATASETS = [
-    "../datasets/karate-disjoint.csv",
-    "../datasets/dolphins.csv",
-    "../datasets/netscience.csv",
+DATASETS = [PurePath(RAPIDS_DATASET_ROOT_DIR)/f for f in [
+    "karate-disjoint.csv",
+    "dolphins.csv",
+    "netscience.csv"]
 ]
+
+
 #            '../datasets/email-Eu-core.csv']
 
 STRONGDATASETS = [
-    "../datasets/dolphins.csv",
-    "../datasets/netscience.csv",
-    "../datasets/email-Eu-core.csv",
+    PurePath(RAPIDS_DATASET_ROOT_DIR)/f for f in [
+        "dolphins.csv",
+        "netscience.csv",
+        "email-Eu-core.csv"]
 ]
 
+
 DATASETS_KTRUSS = [
-    ("../datasets/polbooks.csv", "../datasets/ref/ktruss/polbooks.csv")
+    (str(PurePath(RAPIDS_DATASET_ROOT_DIR)/"polbooks.csv"),
```

Comment:
Rather than convert these to strings here, I'd do the string conversion right before you need to pass it to the function that requires it to be a string, just so things stay consistent as long as possible.  Since this is only used in `test_k_truss_subgraph.py`, and I think Nx is requiring the string, I'd just change the call to be `nx.read_edgelist(str(graph_file), ...)` in `test_k_truss_subgraph.py`

Reply:
you are right. since `nx.read_edgelist `doesn't support `PurePath` it is better to cast its input instead of altering the consistency of the `utils` file.